pom.xml

```diff
@@ -672,10 +672,10 @@
               </reportSet>
             </reportSets>
           </plugin>
-          <plugin>
+          <!--<plugin>
             <groupId>org.revapi</groupId>
             <artifactId>revapi-maven-plugin</artifactId>
-            <version>0.14.6</version> <!-- {x-version-update;org.revapi:revapi-maven-plugin;external_dependency} -->
+            <version>0.14.6</version> &lt;!&ndash; {x-version-update;org.revapi:revapi-maven-plugin;external_dependency} &ndash;&gt;
```

Comment:
is this intentional?

Reply:
No, i just commented this plugin and IDE changed it i guess.

Reply:
i think its how the xml marks comments inside a comment.